### PR TITLE
feat: :sparkles: add types for has-values

### PR DESCRIPTION
**Pull request by Ankan002**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## has-values ([<kbd>unpkg</kbd>](https://unpkg.com/browse/has-values@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'has-values'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d882f043fc9b92990fec241889517ec892030ac8
</p>

File: types/has-values/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// Type definitions for has-values 2.0
+// Project: https://github.com/jonschlinkert/has-values
+// Definitions by: Ankan Bhattacharya <https://github.com/Ankan002>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default function has(value: unknown): boolean;
```
Comment (peterblazejewicz) [inline types/has-values/index.d.ts:6]: this is not allowed, modules should reflect nature of the JS module, here CJS module with export:
https://github.com/jonschlinkert/has-values/blob/master/index.js#L12
The TS conventions and default configuration allows to use CJS exports just like 'export default' exports, not the other way around

**Review by peterblazejewicz — CHANGES_REQUESTED** [review]
Please use CommonJS export and remove exclusion in tslint config

**Comment by Ankan002** [comment]
@peterblazejewicz Thank you for your suggestions, and I have implemented the changes. Please review it once.

**Review by peterblazejewicz — APPROVED** [review]

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@Ankan002 thanks!

**Comment by Ankan002** [comment]
Ready to merge

**Pull request merged**